ml-algorithms/build.gradle

```diff
@@ -41,7 +41,7 @@ dependencies {
 }
 
 configurations.all {
-    resolutionStrategy.force 'com.google.protobuf:protobuf-java:3.21.9'
+    resolutionStrategy.force 'com.google.protobuf:protobuf-java:3.21.7'
```

Comment:
Why protobuf version is downgraded?

Reply:
in github, When I commit code, there will be the dependency check and I changed it according to the tip

Reply:
I also have a PR to ml-commons, and I didn't change this, no error raised: https://github.com/opensearch-project/ml-commons/pull/645

Reply:
I don't know what happen,but I merge the code from main branch. It didn't pass the check ,so I changed it.

Reply:
rollback